### PR TITLE
Insert default hub for 'install' command if it exists

### PR DESCRIPTION
**Pull request by mathetake**
Signed-off-by: Takeshi Yoneda <takeshi@tetrate.io>

### CODE REVIEW

**Comment by mathetake** [comment]
note that this points to #38 and we merge this after it is merged.

**Review by deva26 — APPROVED** [review]
Much appreciated.

**Comment by mathetake** [comment]
OK, corrected the bug while I'm testing this locally, and now fixed. merging!

**Pull request merged**
